src/common/CMakeLists.txt

```diff
@@ -1,15 +1,14 @@
 add_subdirectory(binaries_op)
-add_subdirectory(bzip2_op)
 add_subdirectory(config)
 add_subdirectory(data_provider)
 add_subdirectory(dbsync)
 add_subdirectory(error_messages)
-add_subdirectory(file_op)
 add_subdirectory(filesystem_wrapper)
 add_subdirectory(hashHelper)
-add_subdirectory(logger)
 add_subdirectory(mem_op)
+add_subdirectory(logger)
```

Comment:
It looks like this was sorted alphabetically before, this should go before mem_op

Reply:
Done